custom_components/ojmicroline_thermostat/binary_sensor.py

```diff
@@ -14,28 +14,28 @@
 from .coordinator import OJMicrolineDataUpdateCoordinator
 from .models import OJMicrolineEntity
 
-BINARY_SENSOR_TYPES: dict[str, BinarySensorEntityDescription] = {
-    "online": BinarySensorEntityDescription(
+BINARY_SENSOR_TYPES: list[BinarySensorEntityDescription] = [
```

Comment:
I didn't see an obvious reason for this to be a dict so I just migrated to a list. Let me know if I'm missing context.